rdf_entity.install

```diff
@@ -26,3 +31,51 @@ function rdf_entity_requirements($phase) {
   $requirements += rdf_entity_virtuoso_permission_requirements();
   return $requirements;
 }
+
+/**
+ * Move RDF entity mapping data from bundle entities into dedicated entities.
+ */
+function rdf_entity_update_8001() {
+  $entity_type_manager = \Drupal::entityTypeManager();
+
+  // Update or post-update scripts might need this config entity available when
+  // they run. We don't wait on configuration synchronization, because that runs
+  // usually after the database update, so we make this entity available in an
+  // early stage of updates.
+  $values = Yaml::decode(file_get_contents(__DIR__ . '/config/install/rdf_entity.graph.default.yml'));
+  RdfEntityGraph::create($values)->save();
+
+  // Iterate over all entities that are bundles of content entities with
+  // RdfEntitySparqlStorage and move their 3rd party settings belonging to
+  // rdf_entity module into their dedicated rdf_entity_mapping config entities.
+  foreach ($entity_type_manager->getDefinitions() as $entity_type_id => $entity_type) {
+    if ($storage = $entity_type_manager->getStorage($entity_type_id)) {
```

Comment:
+1 for this. Also `if ($storage = .... ->getStorage(...)) {...}` doesn't make any sense because `::getStorage()` doesn't return anything. It only throws an exception if fails the return the storage.